### PR TITLE
SendUsAMeessageStatus

### PR DESCRIPTION
**Pull request by StetcoClaudiu**

### CODE REVIEW

File: g9frontend/src/components/SendUsAMessageButton/SendUsAMessageButton.tsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+        // Send a POST request to the /developer-message endpoint
+        fetch('/developer-message', {
+            method: 'POST',
+            headers: {
+                'Content-Type': 'application/json',
+            },
+            body: JSON.stringify({ text: messageText }),
+        })
+    };
```
Comment (Andrei-Fabian-Pop) [inline g9frontend/src/components/SendUsAMessageButton/SendUsAMessageButton.tsx:19]: The SendUsAMessageButtonComponent should trigger a text box in which the user can type a message and after that, the box will close and the POST method will be called.

File: g9frontend/src/components/SendUsAMessageButton/SendUsAMessageButtonStyle.css [hunk 2]
```diff
@@ -0,0 +1,8 @@
+.sendUsAMessageButton{
+    color:"black" !important;
+    border-radius: '50%' !important;
+    position: 'fixed' !important;
+    bottom: '20px' !important;
+    right: '20px' !important; 
+    box-shadow: '0px 0px 10px rgba(0, 0, 0, 0.2)' !important;
+}
```
Comment (Andrei-Fabian-Pop) [inline g9frontend/src/components/SendUsAMessageButton/SendUsAMessageButtonStyle.css:8]: 8 rows and 9 errors, plis fix

File: package-lock.json [hunk 3]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (Andrei-Fabian-Pop) [inline package-lock.json:1]: please remove the package-lock.json file from your commits

**Review by Andrei-Fabian-Pop — CHANGES_REQUESTED** [review]

**Comment by bia1708** [comment]
@Andrei-Fabian-Pop can you please confirm my last commit looks alright?

**Review by Andrei-Fabian-Pop — APPROVED** [review]
Looks good

**Pull request merged**